java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/kms/AwsKeyManagementServiceImpl.java

```diff
@@ -229,12 +236,6 @@ GenerateDataKeyResult generateDataKey(final Map<String, AwsKmsArnClient> sortedR
     throw new KmsException("could not successfully generate data key using any regions");
   }
 
-  /**
-   * @param region
-   * @param arn
-   * @param encryptedKeyEncryptionKey
-   * @return
-   */
```

Comment:
can be handled when we get around to adding JavaDocs for non-public methods. since these PRs are focussed only on public methods, this really isn't in the scope of this PR

Reply:
yea, so can just leave it in there rather than removing them now and adding them again later.